invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/InvitationActionButtons.js

```diff
@@ -9,30 +9,50 @@
 import React from "react";
 import { Button } from "semantic-ui-react";
 import { i18next } from "@translations/invenio_communities/i18next";
+import _isEmpty from "lodash/isEmpty";
+import PropTypes from "prop-types";
 
-export const ViewButton = ({ onClick }) => (
-  <Button size="tiny" content={i18next.t("View")} onClick={onClick} />
-);
+export const ViewButton = ({ onClick, show }) =>
```

Comment:
I think it is an overkill to create separate components for these buttons, since they are not reused. They could be placed in the return of the ActionButtons component